.gitignore

```diff
@@ -1,3 +1,4 @@
+jib-*/bin
```

Comment:
What is this referring to?

Reply:
Oh I wasn't supposed to commit that.  I have Eclipse set to generate to bin/, which prevents clashes when running Maven and Gradle from outside.